assembly/src/parsers/io_ops.rs

```diff
@@ -256,69 +276,137 @@ pub fn parse_loc_storew(op: &Token) -> Result<Node, ParsingError> {
 // HELPER FUNCTIONS
 // ================================================================================================
 
-fn parse_constants(op: &Token) -> Result<Vec<Felt>, ParsingError> {
-    let mut constants = Vec::new();
-    let param_idx = 1;
-    let param_count = op.num_parts() - param_idx;
+/// Parses a hexadecimal or decimal parameter values into a vector of u64 values valid for field
+/// element creation and returns appropriate instruction based on parsed values
```

Comment:
nit: would change this to: "Parses a list of parameters (each of which could be in decimal or hexadecimal form) and returns an appropriate push instruction node."